add-ons/istio/istio.addon

```diff
@@ -1,37 +1,45 @@
 # Name: istio
 # Description: Deploys Latest Istio
 # Url: https://github.com/istio/istio
-# Required-Vars: ISTIO_HOME
-# OpenShift-Version: >=3.7.0
-
+# OpenShift-Version: >=3.10.0
+# Depends-On: dynamic-admission-controllers
 # From istio doc
-oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-pilot-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z default -n istio-system
-
-# Create Istio 
-oc create -f #{ISTIO_HOME}/install/kubernetes/istio.yaml
 
-oc expose svc istio-ingress -n istio-system
+# TODO https://github.com/istio/istio/issues/4500
+echo Disable SELinux (https://github.com/istio/istio/issues/4500)
+ssh sudo setenforce 0
```

Comment:
@literalice Does this still not resolve in the 3.10 because that is something will be deal breaker for SELinux enabled systems, minishift also by default have it enforcing.

Reply:
@praveenkumar Thank you for pointing it out. As information from @kameshsampath , it is a regression of istio v1.0.1.
I've used v1.0.0 and confirmed that istio sidecar auto-injection & knative serving work well with SELinux.